_static/images/coala_logo.svg

```diff
@@ -14,8 +14,7 @@
    height="48px"
    id="svg2985"
    version="1.1"
-   viewbox="0 0 48 48"
-   inkscape:version="0.91 r13725"
+   inkscape:version="0.92.3 (2405546, 2018-03-11)"
```

Comment:
see https://github.com/coala/documentation/pull/558 and https://github.com/coala/artwork/pull/33